src/Persistent/Idiom.hs

```diff
@@ -157,13 +156,8 @@ saveAndEnactFreeze :: Timestamp -> Freeze -> AUpdate ()
 saveAndEnactFreeze now shouldBeFrozenOrNot = do
   dbFreeze .= shouldBeFrozenOrNot
   let change = case shouldBeFrozenOrNot of
-          NotFrozen -> PhaseThaw   now
-          Frozen    -> PhaseFreeze now
-      freezeOrNot topic = do
-          case phaseTrans (topic ^. topicPhase) change of
-            Nothing -> return ()  -- ignored, probably repeated freeze or thaw
-            Just (phase', actions) -> do
-                let !_A = assert (null actions) ()  -- wrong monad to execute'em
-                setTopicPhase (topic ^. _Id) phase'
```

Comment:
so, the assertion is now eliminated?  i think i'll try to re-add it, hope that's ok?